libexec/erlang

```diff
@@ -1157,6 +1145,20 @@ __show_upgrade_version_does_not_differ_error_message() {
     error_message "for automatic upgrades by setting AUTO_VERSION=revision in the config."
 }
 
+__find_remote_s3cmd() {
```

Comment:
I would this is more of a check, then search. Maybe `__check_remote_s3cmd` or `__ensure_remote_s3_cmd` or  something alike.

Reply:
__ensure_remote_s3_cmd sounds good, will rename it